lib/benchmark/CMakeLists.txt

```diff
@@ -160,9 +160,12 @@ else()
   add_cxx_compiler_flag(-Wall)
   add_cxx_compiler_flag(-Wextra)
   add_cxx_compiler_flag(-Wshadow)
-  add_cxx_compiler_flag(-Werror RELEASE)
-  add_cxx_compiler_flag(-Werror RELWITHDEBINFO)
-  add_cxx_compiler_flag(-Werror MINSIZEREL)
+  # Disable -Werror because of gcc bug
```

Comment:
I think it's ok to simply inline the patch here. It's only a workaround that should hopefully become obsolete in the future.

Reply:
I don't quite understand. So its fine as is or should I also commit a (imo redundant) patch file?

Reply:
Not sure how soon this will be fixed though. Neither google seems to make any effort working around this issue, nor the gcc team sees that as very urgent...

Reply:
All fine. Hopefully, we don't need to update the lib anytime soon.